manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0

```

Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```